std_msgs/Header header
GPnetGrasp[] grasps
